public/index.php

```diff
@@ -1,81 +1,87 @@
 <?php declare(strict_types=1);
 
 use Shopware\Production\HttpKernel;
-use Symfony\Component\Dotenv\Dotenv;
-use Symfony\Component\ErrorHandler\Debug;
 use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+use Symfony\Component\HttpKernel\TerminableInterface;
 
-if (\PHP_VERSION_ID < 70400) {
-    header('Content-type: text/html; charset=utf-8', true, 503);
+$_SERVER['SCRIPT_FILENAME'] = __FILE__;
 
-    echo '<h2>Error</h2>';
-    echo 'Your server is running PHP version ' . \PHP_VERSION . ' but Shopware 6 requires at least PHP 7.4.0';
-    exit(1);
+require_once __DIR__ . '/../vendor/autoload_runtime.php';
+
+if (!file_exists(__DIR__ . '/../.env')) {
+    $_SERVER['APP_RUNTIME_OPTIONS']['disable_dotenv'] = true;
```

Comment:
The runtime will automatically load the .env file. However, if it does not exist, it would look for a .env.dist file. This file is not meant to be loaded, so we prevent DotEnv if no .env file exists.